opensrp-app/src/test/java/org/smartregister/repository/PlanDefinitionRepositoryTest.java

```diff
@@ -130,6 +132,21 @@ public void testFindPlanDefinitionByIdShouldReturnNull() {
                 new String[]{"4708ca0a-d0d6-4199-bb1b-8701803c2d02"});
     }
 
+    @Test
+    public void testFindAllPlanDefinitions() {
```

Comment:
Why is the test mocking the method under test?

Reply:
The test has been refactored